README.md

```diff
@@ -19,7 +19,16 @@ Also, if you are an "autodidact" programmer or atleast aspire to be one, this co
 - Export the `cmd.py` using the `stackdiadact` alias either by adding the line below to your `.bashrc` or `.bash_aliases`:
 `alias stackdidact="python3 path/to/stackdidact/cmd.py"`
 - Restart your terminal emulator or start a new shell.
-- Type `stackdidact`.
+
+## Running
```

Comment:
I had a different idea about this.
Instead of asking the user to give command line arguments beforehand for browsing to a discussion, I think a redirect to the above said question *after* `stackdidact` is executed will seem fitting.